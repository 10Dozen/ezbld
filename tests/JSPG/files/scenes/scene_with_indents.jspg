# SceneWithIndents

    *goto:    "Scene2"   

              *pre_exec:     Foo.Bar()    
    
Desc line 1. 

   Desc line 2.   
       
         Desc line 3.