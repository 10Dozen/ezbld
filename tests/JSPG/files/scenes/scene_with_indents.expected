[0]
cls = JSPGScene
name = SceneWithIndents
goto = "Scene2"
pre_exec = Foo.Bar()
desc =
    Desc line 1.

    \tDesc line 2.

    \t\tDesc line 3.